deployment/geotrellis-cluster.config.example

```diff
@@ -0,0 +1,13 @@
+[CONFIG NAME]
```

Comment:
This is the example config file that is going to do most of the driving of managing stacks. The intention here is that you can have configs for different stacks and use `./gt-stack.py` to manage them separately by specifying an optional parameter to the `./gt-stack.py` command.